monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md

```diff
@@ -0,0 +1,30 @@
+# Conheça seus hard forks
+
+A não ser que você seja desenvolvedor, o termo "hard fork" pode causar muita confusão e potencialmente trazer conotações negativas para você. Afinal, você já deve ter lido um artigo ou outro sobre hard forks e como ele causou contenção ou drama em alguns projetos de cripto. Os hard forks do Monero acontecem de maneira interessante e enquadram-se em três categorias principais:
+
+### 1) Atualização da rede / Hard fork regularmente programado
+
+Os desenvolvedores do Monero têm por objetivo realizar dois hard forks regulares por ano. Esses hard forks são sempre feitos com o consenso da comunidade e liderado pelos Desenvolvedores Core. Eles geralmente incluem diversas melhorias ao código principal do Monero. É necessário que você, como usuário Monero, atualize suas carteiras sempre que isso ocorrer e que não é responsabilidade de ninguém, senão você mesmo, de fazê-lo. Ninguém disse que seria fácil ser um pioneiro da tecnologia, mas se for reparar bem, não é tão difícil também.
+
+Outros projetos utilizaram os hard forks regularmente programados do Monero para seu próprio benefício e criaram seus próprios projetos sobre as cadeias _(chains)_ anteriores do Monero. Esses projetos costumam fracassar rapidamente, levando consigo a ganância dos seus criadores. Se você não atualizar sua carteira Monero após um desses hard forks e transferir seus XMR, você pode acabar perdendo-os em uma _chain_ diferente ou gastar um tempo considerável tentando recuperá-los.
+
+_Nível de segurança:_ A melhor prática é atualizar suas carteiras Monero assim que atualizações forem lançadas.
```

Comment:
Maybe it is interesting to add "atualizações de hard forks" on the end. So it enphasizes which updates you are talking about.

Reply:
I would say that users should update their software either being it due a hard fork or not. What do you think?

Reply:
Agreed.